js/contacts.js

```diff
@@ -47,52 +51,15 @@ $.ajax({
                         </div>`;                  
                 var element = document.getElementById("test");
                 element.innerHTML = result;
-                $('#test_error').show();
-
+                $('#test').show();
```

Comment:
This looks like an unintentional revert and unrelated to "add federated share type to sciencemeshShareProviderFactory"?

Reply:
I check the UI html code and other was not any tag with "test_error" id. so I concluded that I should remove first line.

Reply:
@parhamin2010 what is the correct element name now?

Reply:
It's id = test